src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -464,6 +464,7 @@ Async.CombinedBuilder<FieldValueInfo> getAsyncFieldValueInfo(
             QueryDirectives queryDirectives = new QueryDirectivesImpl(field,
                     executionContext.getGraphQLSchema(),
                     executionContext.getCoercedVariables().toMap(),
+                    executionContext.getNormalizedVariables().toMap(),
```

Comment:
needed inside to make normalized values where they refeference operation variables